lib/licenses.nix

```diff
@@ -817,14 +817,54 @@ lib.mapAttrs (n: v: v // { shortName = n; }) {
   };
 } // {
   # TODO: remove legacy aliases
```

Comment:
I think this todo comment ca be removed.

Reply:
We still want to get rid of these to prevent people from using the unclear licenses for new packages.